modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -849,7 +917,8 @@ private function add_style_tab(): void {
 				'label'     => esc_html__( 'Text Color', 'elementor' ),
 				'type'      => Controls_Manager::COLOR,
 				'selectors' => [
-					'{{WRAPPER}} .e-link-in-bio' => '--e-link-in-bio-ctas-text-color: {{VALUE}}',
+					'{{WRAPPER}} .e-link-in-bio'                   => '--e-link-in-bio-ctas-text-color: {{VALUE}}',
+					'{{WRAPPER}} .e-link-in-bio__cta.is-type-link' => '--e-link-in-bio-ctas-text-color: {{VALUE}}',
```

Comment:
```suggestion
					'{{WRAPPER}}'  => '--e-link-in-bio-ctas-text-color: {{VALUE}}',
```
Does it work if you just target the wrapper? When using CSS variables in general, I think we just need the `{{WRAPPER}}`

Reply:
@willhowat told me that this is not working if he only targets the wrapper, correct @willhowat ?